src/ontodia/data/sparql/sparqlDataProviderSettings.ts

```diff
@@ -67,6 +73,14 @@ export interface SparqlDataProviderSettings {
      * imposes additional filtering on elements within filter
      */
     filterAdditionalRestriction: string;
+
+    /**
+     * Link reference resolving. Developer can use it to transform how link navigation is performed.
+     */
+    refElementLinkQueryOut: string; // '\${refElementIRI} \${refElementLinkIRI} ?inst . FILTER ISIRI(?inst)',
```

Comment:
You should move the default values from inline comments to JSDoc `@default` delcaration.